tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>FYI. In the same way that <code>tf.keras</code>  points to the API docs automatically, it seems like the same will be true for tfmot (e.g. <code>tfmot.clustering.keras.cluster_weights</code>) . I haven't seen this for myself, but will be checking for the quantization launch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Verified it is the case. See https://www.tensorflow.org/model_optimization/guide/quantization/training_comprehensive_guide.md

Reply:
<p>Done, thanks.</p>